src/tr/expand.rs

```diff
@@ -88,14 +88,19 @@ impl<'a> Iterator for ExpandSet<'a> {
 
         if let Some(first) = self.unesc.next() {
             // peek ahead
-            if self.unesc.peek() == Some(&'-') && match self.unesc.size_hint() {
-                (x, _) if x > 1 => true, // there's a range here; record it in our internal Range struct
-                _ => false,
-            } {
+            if self.unesc.peek() == Some(&'-')
+                && match self.unesc.size_hint() {
```

Comment:
Rather than using `match`, change the second condition to something like `self.unesc.size_hint().0 > 1`.

Reply:
done